### ISSUE TITLE
Create DOE Prototype Building BCL measure creates a building where the exterior roof material is drywall

### ISSUE THREAD

**Issue by mldichter** [NREL/openstudio-standards#545]
FullServiceRestaurant

![roof name](https://user-images.githubusercontent.com/39242689/49113630-e5b2c480-f24a-11e8-81e5-f887781529f9.PNG)

BuildingSurface:Detailed,
    west-roof,               !- Name
    Roof,                    !- Surface Type
    Typical Wood Joist Attic Floor R-37.04,  !- Construction Name
    attic ZN,                !- Zone Name
    Outdoors,                !- Outside Boundary Condition
    ...

Construction,
    Typical Wood Joist Attic Floor R-37.04,  !- Name
    5/8 in. Gypsum Board,    !- Outside Layer
    Typical Insulation R-35.4;  !- Layer 2

**Comment by mdahlhausen** [comment]
Could not recreate issue.

**Comment by mldichter** [comment]
I was also unable to recreate the Full Service Restaurant idf with the drywall roof with the current version of the measure.

I went searching through my recycle bin to find the file, and I did. Looks like this was fixed at some point and the issue never closed.

I submitted a few issues related to the Create DOE Prototype Building BCL measure. Maybe other issues were fixed and can be closed with a quick check.
